dd-sdk-android-tv/README.md

```diff
@@ -0,0 +1,76 @@
+# Datadog Integration for Android TV applications
+
+## Getting Started 
+
+To include the Datadog integration for AndroidTv in your project, simply add the
+following to your application's `build.gradle` file.
+
+```groovy
+dependencies {
+    implementation "com.datadoghq:dd-sdk-android:<latest-version>"
+    implementation "com.datadoghq:dd-sdk-android-tv:<latest-version>"
+}
+```
+
+### Initial Setup
+
+Before you can use the SDK, you need to setup the library with your application
+context, your Client token and your Application ID. 
+To generate a Client token and an Application ID please check **UX Monitoring > RUM Applications > New Application**
+in the Datadog dashboard.
+
+To get more information in the RUM Action events for AndroidTV application you can provide the [LeanbackViewAttributesProvider][1]
+when initializing the SDK.
+
+```kotlin
+    class SampleApplication : Application() {
+        override fun onCreate() {
+            super.onCreate()
+            val configuration = Configuration.Builder(
+                    logsEnabled = true,
+                    tracesEnabled = true,
+                    crashReportsEnabled = true,
+                    rumEnabled = true
+                )
+                .useSite(DatadogSite.US3)
```

Comment:
why are you showing the US3 site?

Reply:
It is just copy pasted from the others `README`, going to replace that.